samples/net/wifi/boards/cy8cproto_062_4343w.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_NET_IF_MAX_IPV4_COUNT=2
+CONFIG_NET_MGMT_EVENT_QUEUE_TIMEOUT=20
+# Debugging
+CONFIG_WIFI_LOG_LEVEL_DBG=y
+CONFIG_DEBUG=y
```

Comment:
Most board overlays do not enable that, so maybe let's remove it and leave it to the user. Or do you see value in enabling those by default?

Reply:
removed.